src/index.html

```diff
@@ -3,10 +3,384 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home">
+        <img class="logo"
+        src="./images/logo.png"
+        alt="Moyologo">
```

Comment:
please check the code styling
```suggestion
        <img
          class="logo"
          src="./images/logo.png"
          alt="Moyologo"
        >
```